README.md

```diff
@@ -66,6 +66,14 @@ docker compose up
 
 After the build is complete you should be able to visit the application on `localhost:3000`
 
+#### 4. Add data to be able to create a game and invite player
```

Comment:
This makes sense.  I was just making a second account in an incognito browser and creating games that way